crates/air_r_formatter/src/r/auxiliary/call_arguments.rs

```diff
@@ -1239,11 +1221,25 @@ fn argument_is_groupable(argument: &AnyRExpression) -> SyntaxResult<bool> {
         //   }
         // )
         // ```
-        RFunctionDefinition(_) => true,
+        RFunctionDefinition(node) => node.body().is_ok_and(|body| match body {
+            RBracedExpressions(ref body) => braced_expressions_is_groupable(body, comments),
+            _ => true,
+        }),
 
         // Nothing else benefits from grouping
         _ => false,
     };
 
     Ok(result)
 }
+
+fn braced_expressions_is_groupable(node: &RBracedExpressions, comments: &RComments) -> bool {
+    match braced_expressions_variant(node, comments) {
+        // These two stay collapsed as `{}` or `{{ x }}`, and don't benefit from grouping
+        BracedExpressionsVariant::Empty => false,
+        BracedExpressionsVariant::CurlyCurly(_) => false,
+        // These two expand across multiple lines, and benefit from grouping
+        BracedExpressionsVariant::EmptyWithDanglingComments => true,
+        BracedExpressionsVariant::NotEmpty => true,
+    }
+}
```

Comment:
Reuse of the `braced_expressions_variant()` helper to allow us to easily decide if we should consider using best fitting on this trailing argument or not